h/search/persistence.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+"""Code for persisting (indexing) annotations to Elasticsearch."""
+
+from __future__ import unicode_literals
+
+import elasticsearch_dsl
+
+
+class Annotation(elasticsearch_dsl.DocType):
+    """An elasticsearch_dsl annotation document."""
+
+    class Meta:
+        index = "hypothesis"
+
+    authority = elasticsearch_dsl.Keyword(index=False)
+
+    @staticmethod
+    def create(models_annotation):
+        """Return a new search.Annotation from the given models.Annotation."""
+        return Annotation(meta={"id": models_annotation.id})
```

Comment:
With this being a new file I was able to format it with <https://github.com/ambv/black> :)

Reply:
The `create()` method is because I don't think it's sage to override `__init__()`. And it's also where we can do things based on the annotation model object, such as using its id as the Es document id. Much of what `AnnotationSearchIndexPresenter` does will end up in this `create()` I think